deploy/helm/charts/values.yaml

```diff
@@ -118,7 +120,10 @@ helperPod:
     repository: openebs/linux-utils
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: 4.0.0
+    tag: 4.1.0
```

Comment:
Was there any change to linux-utils?

Reply:
We created a new tag for the alpine-bash and alpine-sh which also created a new linux-utils but no change in code.

Reply:
No, there wasn't but we use the latest one usually.

Reply:
Ahh right, the base image is changed. So it's good to have this.